process/charter.md

```diff
@@ -3,78 +3,161 @@ title: How to create a Working Group or Interest Group
 toc: yes
 ---
 
-The W3C Process describes the [lifecycle of chartered groups](https://www.w3.org/policies/process/#group-lifecyle). At a high level, W3C approves a new Working Group or Interest Group charter after a series of reviews intended to improve the quality of the charter and generate interest in the work. The reviews typically happen in this order:
+The W3C Process describes the [lifecycle of chartered groups](https://www.w3.org/policies/process/#group-lifecyle).
 
-- Early socialization (e.g., by the Team or in a Community Group)
-- Review by the W3C Strategy Team
-- More socialization within the community
-- W3C "Horizontal" Group review
-- Advisory Committee review
-- In the case of [Formal Objections](../council/council.md), W3C Council review
+**Note:** [Group closures](closing-wg.md) are addressed in other documentation.
 
-In this document we describe the operational aspects of these reviews.
+## 1. Overview of charter development
 
-*Timing:* Please note that these reviews take time, and people working on charters should expect the entire process to take multiple months.
+W3C creates charters for Working and Interest Groups based on W3C community interest. W3C fosters awareness of charters, improves their quality, and gauges Membership support in phases:
 
-**Note:** [Group closures](closing-wg.md) are addressed in other documentation.
+* **Preparation** 
+* **Charter refinement** to help ensure charters are mission-aligned, reflect community input and consensus, and have been well-socialized and widely reviewed.
+* **Formal review by the W3C Advisory Committee**, followed by a W3C Decision (including handling of any [Formal Objections](../council/council.md)).
 
-## 1. New groups  {#new-charters}
+In this document we describe the operational aspects of these phases.
 
-Discussions for new work happen in a variety of venues, including Member discussions, Workshops, Working / Interest / Community Groups, and the Team.
+**Note**: Organizing discussions and reviews, building consensus, and handling of formal objections all require time. People working on charters should expect the entire process to take multiple months.
```

Comment:
For consistency (following Process 2025): s/formal objections/Formal Objections/